src/style.css

```diff
@@ -0,0 +1,129 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+  
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+  
+html,
+* {
+  box-sizing: border-box;
+  font-family: "Avenir", sans-serif;
+}
+
+.bigForm {
+  position: relative;
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big_img {
+  position: absolute;
+  display: block;
+  width: 19px;
+  height: 19px;
+  left: 26px;
+  top: 25px;
+}
+
+.input_big {
+  height: 68px;
+  width: 100%;
+  outline: none;
+  border: none;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-left: 62px;
+}
+
+.input_big::placeholder {
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  padding-top: 25px;
+  padding-bottom: 23px;
+}
+
+.input_big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input_big:focus {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  color: #3d4e61;
+  filter: drop-shadow(0 4px 4px rgba(0, 0, 0, 0.25));
+  font-weight: 900;
+}
+
+.smallForm {
+  display: block;
+  position: relative;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-top: 20px;
+}
+  
+.input_small {
+  width: 100%;
+  height: 40px;
+  border: none;
+  outline: none;
+  padding-left: 33px;
+  font-weight: 300;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+}
```

Comment:
you define a lot of similar styles for both inputs, please add some class and use it for both
use DRY principle (Don't Repeat Yourself)